### ISSUE TITLE
Remove "scripts" dir now that we don't do per-module analyses

### ISSUE THREAD

**Issue by alilleybrinker** [mitre/hipcheck#350]
The one script in the "scripts" directory was for per-module analysis support. Since we no longer do that, we can remove it. This also moves us closer to eliminating the need for a post-install step. Right now, our post-install `setup` command copies over this one script, plus a bundle of configuration files. With the move to the policy file concept, one of my goals is to reduce the need for all of these additional files, instead centralizing configuration in the KDL file (and having it be optional with defaults), so I think we can get to a place where `setup` isn't _required_, or at least is just used to generate a `Hipcheck.kdl` starter file in your current directory

**Comment by mchernicoff** [comment]
Should we remove the per-module analysis entirely at the same time? We still have that metric and some supporting code in Hipcheck. Since removing `module-deps.json` could break that code, it might be bad to leave it in. On the other hand, that code is not called, and I presume there will be a general pruning of the old style analyses at some point soon anyway.

**Comment by alilleybrinker** [comment]
Yeah we can remove it. If we ever want to go back and use that again it's in the Git history.
